### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by anupkgurung**

### CODE REVIEW

File: css/navbar.css [hunk 1]
```diff
@@ -1,8 +1,4 @@
-/* body {
```
Comment (Naveenchandar) [inline css/navbar.css:1]: Please remove commented code, if its not used.

File: css/list.css [hunk 2]
```diff
@@ -26,3 +26,12 @@ ul {
     background-color: var(--doc-hover-light);
     border-radius: 5px;
 }
+.list-item-float {
+    float: right;
```
Comment (Naveenchandar) [inline css/list.css:30]: Why float ? Any specific reason ?

Comment (anupkgurung) [reply css/list.css:30]: just to float list item to right.

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Review by ishitakeshawani — APPROVED** [review]
LGTM

**Pull request merged**